src/internal/hooks/use-visual-mode/__tests__/use-visual-mode.test.tsx

```diff
@@ -4,6 +4,12 @@ import React from 'react';
 import { useVisualRefresh, clearVisualRefreshState } from '../index';
 import { render, screen } from '@testing-library/react';
 
+declare global {
```

Comment:
I am guessing it is safe, because it is test-only code, but can we use the same `ExtendedWindow` approach here as well, for consistency?